index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
This looks very cool - however I would suggest that the emojis added are confirmed to run in/on every browser/device type, otherwise this could cause a visually undesirable effect.